### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by yang-zj1026**
update readme and training script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ python -m swarm_rl.train --env=quadrotor_multi --train_for_env_steps=1000000000
 --quads_collision_hitbox_radius=2.0 --quads_collision_falloff_radius=4.0 --quads_local_obs=6 \
 --quads_local_metric=dist --quads_local_coeff=1.0 --quads_num_agents=8 --quads_collision_reward=5.0 \
 --quads_collision_smooth_max_penalty=10.0 --quads_neighbor_encoder_type=attention \
---replay_buffer_sample_prob=0.75 --anneal_collision_steps=300000000 --experiment=swarm_rl 
+--replay_buffer_sample_prob=0.75 --anneal_collision_steps=300000000 --experiment=swarm_rl \
```
Comment (Zhehui-Huang) [inline README.md]: Could you tell me why we need this?

**Review by Zhehui-Huang — CHANGES_REQUESTED** [review]
Overall, it is a good job! There is only one minor thing I need your clarification, please check my comment.

File: README.md (hunk 1 above)
Comment (yang-zj1026) [reply README.md]: ah I think it's just a redundant backslash I've typed, feel free to remove it.

**Review by Zhehui-Huang — APPROVED** [review]
Good!

**Pull request merged**